src/Discord/Parts/User/Client.php

```diff
@@ -42,7 +42,23 @@ class Client extends Part
     /**
      * @inheritdoc
      */
-    protected $fillable = ['id', 'username', 'email', 'verified', 'avatar', 'discriminator', 'bot', 'user', 'application'];
+    protected static $fillable = ['id', 'username', 'email', 'verified', 'avatar', 'discriminator', 'bot', 'user', 'application'];
+
+	/**
+     * Returns the fillable attributes.
+     *
+     * @return array
+     */
+    public static function getFillableAttributes($context = '')
```

Comment:
Not sure why the function is duplicated here? Also duplicated in `Application`.

Reply:
Polymorphism, it overrides the function of the same name on the Part class

Reply:
Is it not the same function?

Reply:
This function currently uses the self keyword to refer to $fillable, so this function would get the fillable attributes of the parent class Part, which is incorrect behavior.